package io.github.hankaviator.gdialertweak;

import android.app.Application;

import com.google.android.material.color.DynamicColors;

public final class GDialerApplication extends Application {
    @Override public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}
